Dshop/apps/products_catalogue/tests/test_persistent_cart.py

```diff
@@ -6,6 +6,21 @@
 from dj_shop_cart.cart import get_cart_class
 
 
+@pytest.fixture
+def fixture_cart(tv_product, edifier_product, fake_cart_detail_view_request):
+    Cart = get_cart_class() 
+    cart = Cart.new(fake_cart_detail_view_request)
+    cart.add(tv_product, quantity=2)
+    cart.add(edifier_product, quantity=3)
+    return cart
+
+
+@pytest.mark.django_db
```

Comment:
This is a good test for making yourself familiar with the project and tools in the project, but also should be removed. 
The cart is already extensively tested

Reply:
Will be removed